eax/random_vectors_generate_script.go

```diff
@@ -0,0 +1,78 @@
+package eax
+
+import (
+	"crypto/aes"
+	"crypto/rand"
+	"fmt"
+	"io"
+	mathrand "math/rand"
+	"os"
+	"testing"
+	"time"
+)
+
+// The following scripts create random test vectors for the eax package
+// and writes them to a file.
```

Comment:
I'm not sure it's worth committing this file. It's unlikely that anyone will ever run it again, and from the output of `random_vectors.go`, it's already relatively obvious how it was generated.
Also, having this file here adds some imports and functions which might slightly increase the build size and load time of applications that use this library.
(Similarly for the ocb one.)

Reply:
+1

Reply:
Thanks for the thorough reviews